dist/modularity/init.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -ueo pipefail
+
+[[ -f .venv/bin/activate ]] || virtualenv -q -p python3 .venv
+chmod 755 .venv/bin/activate
+
+. .venv/bin/activate
+
+REPO_URL="https://github.com/wiredtiger/layercparse.git"
+BRANCH="main"
+HASH_FILE=".layercparse_commit_hash"
+LATEST_HASH=$(git ls-remote "$REPO_URL" "$BRANCH" | awk '{print $1}')
```

Comment:
Last thing here: time-limit this call

Reply:
updated [706a3ad](https://github.com/wiredtiger/wiredtiger/commit/706a3ad2b138912166d424a10d6fc4448b78601e)